### ISSUE TITLE
Add cap weight calculation and individual path ensemble caps

### ISSUE THREAD

**Issue by dz24** [infretis/infretis#6]
Right now we do not calculate the path weight correctly for the WF move if we use interface_cap. There we should fix it asap.

**Comment by TitusvE** [comment]
@dz24 : Is this issue still open? I assume it has been fixed, no?

**Comment by dz24** [comment]
Yes, you are right. I believe it was added in this PR, https://github.com/infretis/infretis/pull/32. I also recently did a comparison between WF and WF cap for double well and got comparable results.
